src/Paramore.Brighter.Transformers.AWS.V4/Paramore.Brighter.Transformers.AWS.V4.csproj

```diff
@@ -13,15 +13,12 @@
     <ProjectReference Include="..\Paramore.Brighter\Paramore.Brighter.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="System.Text.Json" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="AWSSDK.S3" VersionOverride="$(AWSSDKV4)" />
     <PackageReference Include="AWSSDK.SecurityToken" VersionOverride="$(AWSSecurityTokenV4)" />
     <PackageReference Include="Microsoft.Extensions.Http" />
     <PackageReference Include="Polly.Contrib.WaitAndRetry" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
I don't think we need to explicitly include System.Text.Json unless its nestandard2.0 and we may get warnings about trimming